content/resources/chapter-5/verify-signature-5.tsx

```diff
@@ -68,15 +57,16 @@ function verify(sig_r, sig_s, pubkey_x, pubkey_y, msg) {
 
     return x1;
   }
-  // Implement ECDSA!
-  //   u1 = m / s mod n
-  //   u2 = r / s mod n
-  //   R = G * u1 + A * u2
-  //   r == x(R) mod n
   const sig_s_inverted = invert(sig_s);
+  // Implement ECDSA!
+  //   u1 = msg * s mod n
   const u1 = (msg * sig_s_inverted) % ORDER;
+  //   u2 = r * s mod n
```

Comment:
```suggestion
  //   u2 = r * s^(-1) mod n
```

Same thing as previous comment above